modules/ml/src/main/java/org/apache/ignite/ml/genetic/TruncateSelectionTask.java

```diff
@@ -44,9 +44,9 @@
 /**
  * Responsible for performing truncate selection.
  */
-
 public class TruncateSelectionTask extends ComputeTaskAdapter<List<Long>, Boolean> {
 
```

Comment:
empty line here and please format methods getChromosome and getEnhancedPopulation

Reply:
And TODO in reduce method

Reply:
ybabak, Please share with me what respective tools you are using to validate code style.  Presently, I am using IntelliJ's format command to apply the respective style using 'ignite_codeStyle.xml'

IE:

format -s <PathToIgniteCodeStyle>\ignite_codeStyle.xml -m *.java -r <PathToIgniteCode>\modules\ml\src\main\java\org\apache\ignite\ml\genetic